packages/salesforcedx-vscode-core/src/commands/baseDeployCommand.ts

```diff
@@ -59,20 +59,22 @@ export abstract class BaseDeployExecutor extends SfdxCommandletExecutor<
     execution.processExitSubject.subscribe(async exitCode => {
       this.logMetric(execution.command.logName, startTime);
       try {
-        const deployParser = new ForceDeployResultParser(stdOut);
-        const errors = deployParser.getErrors();
-        if (errors && !deployParser.hasConflicts()) {
-          channelService.showChannelOutput();
-          handleDiagnosticErrors(
-            errors,
-            workspacePath,
-            execFilePathOrPaths,
-            BaseDeployExecutor.errorCollection
-          );
-        } else {
-          BaseDeployExecutor.errorCollection.clear();
+        if (stdOut !== '') {
```

Comment:
can we make this a positive check instead?
ie. `if (stdOut) { doSomething(); }`

Reply:
Yes, we can make it a positive check. Solved!!! Thanks!!!